contracts/Comptroller.sol

```diff
@@ -15,38 +15,6 @@ import "./Governance/AccessControlManager.sol";
  * @author Compound
  */
 contract Comptroller is Ownable2StepUpgradeable, ComptrollerV1Storage, ComptrollerInterface, ExponentialNoError {
-    struct LiquidationOrder {
-        VToken vTokenCollateral;
-        VToken vTokenBorrowed;
-        uint256 repayAmount;
-    }
-
-    struct AccountLiquiditySnapshot {
-        uint256 totalCollateral;
-        uint256 weightedCollateral;
-        uint256 borrows;
-        uint256 effects;
-        uint256 liquidity;
-        uint256 shortfall;
-    }
-
-    struct RewardSpeeds {
-        address rewardToken;
-        uint256 supplySpeed;
-        uint256 borrowSpeed;
-    }
-
-    uint256 internal constant NO_ERROR = 0;
-
-    // closeFactorMantissa must be strictly greater than this value
-    uint256 internal constant closeFactorMinMantissa = 0.05e18; // 0.05
-
-    // closeFactorMantissa must not exceed this value
-    uint256 internal constant closeFactorMaxMantissa = 0.9e18; // 0.9
-
-    // No collateralFactorMantissa may exceed this value
-    uint256 internal constant collateralFactorMaxMantissa = 0.9e18; // 0.9
-
     // PoolRegistry, immutable to save on gas
     /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
     address public immutable poolRegistry;
```

Comment:
According to VEN-967, shouldn't we move also these variables (the rest ones: poolRegistry, accessControl, rewardsDistributors, rewardsDistributorExists) to the ComptrollerV1Storage contract

Reply:
Hey @chechu ,

The rest ones you mentioned are immutable and private variables which can't be moved to the base contract `ComptrollerV1Storage`.